src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -91,6 +96,29 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
 
     componentDidMount(){
 
+        this.mountOverlappingStudiesWarning();
+
+    }
+
+    private mountOverlappingStudiesWarning(){
```

Comment:
this is necessary because we need to mount outside tabs

Reply:
so this warning happens on the results page, the other one happens during study selection?

Reply:
correct